docs/manage/configuration.md

```diff
@@ -355,52 +319,8 @@ Configuring `viam-server` with the Viam app allows you to make use of the cloud
 - [Data Management](/manage/data/)
 - [Machine Learning](/manage/ml/)
 
-A locally configured robot, will not be able to access Viam's cloud features.
-If you are configuring a robot that can never connect to the internet, you can manually create a local full raw JSON config file on your robot.
-
-You can write the contents of this config file manually, or you can use the Viam app's config builder (without connecting your robot to it).
-Once you finish configuring your robot in the **Builder** mode, switch to **Raw JSON** and copy the entire raw JSON into your config file.
-Save the file at <file>/etc/viam.json</file> or another location.
-
-Then start `viam-server`:
-
-{{< tabs >}}
-{{% tab name="Linux" %}}
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sudo viam-server
-```
-
-{{% /tab %}}
-{{% tab name="macOS" %}}
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-viam-server
-```
-
-{{% /tab %}}
-{{< /tabs >}}
-
-If you have stored your config file at a location different from the default location (<file>/etc/viam.json</file>), pass in the path to the config:
-
-{{< tabs >}}
-{{% tab name="Linux" %}}
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sudo viam-server -config <path-to-config>.json
-```
-
-{{% /tab %}}
-{{% tab name="macOS" %}}
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-viam-server -config <path-to-config>.json
-```
-
-{{% /tab %}}
-{{< /tabs >}}
-
-To run `viam-server` as a system service, see [Start automatically on boot](/installation/#start-automatically-on-boot).
+However, if you are configuring a robot that can never connect to the internet, you can manually create a [local configuration file](/appendix/local-configuration-file/) on your robot.

```

Comment:
```suggestion
However, if you are configuring a robot that can never connect to the internet, you can create a [local configuration file](/appendix/local-configuration-file/) on your robot.
```
I don't like the word manually here because you don't ahve to manually do it. You can use the app and then copy

Reply:
Gotcha. I need "manually" for non-app local config (`## Build a local configuration file manually`), but the link from this page should indeed use the generic (choice of in-app or manually not until local-configuration-file.md). 👍